doc/governance/project-docs/GOVERNANCE.md

```diff
@@ -0,0 +1,55 @@
+# Governance Policy
+
+The "Project" is herein defined as the activities related to this specific GitHub repository [`HoloViz`](https://github.com/holoviz/holoviz), within the `HoloViz` GitHub Organization. This purpose of this repository is primarily to host the development activities of the HoloViz.org website. This repository also hosts the HoloViz Organization governance documents, that apply to the wider HoloViz ecosystem.
+
+This document provides the governance policy for the Project. Members agree to this policy and to abide by all Project polices, including the [code of conduct](../org-docs/CODE-OF-CONDUCT.md), [trademark policy](../org-docs/TRADEMARKS.md), and [antitrust policy](../org-docs/ANTITRUST.md) by adding their name to the [MEMBERS.md file](./MEMBERS.md).
+
+## 1. Project Member Roles.
+
+This project may include the following roles. Additional roles may be adopted and documented by the Project. All member role terms may be for life, as long as the members adhere to the Organization's policies.
```

Comment:
This language is sufficiently broad as to allow a BDFL role (or really anything else) at the project level. Is this intended?

Reply:
Flexibility at the Project level was the intent. But of course, some Project governance structures will mesh better with the proposed Org governance than others until we adequately adjust things. Now that I have a better idea of your preferences I can continue adjusting things accordingly